WLC/benchmark.py

```diff
@@ -48,6 +48,11 @@ def run_benchmarks():
         total_accuracy += accuracy * length
         total_length += length
 
+    for i in range(6, 9):
```

Comment:
I don't think that we need this explicit loop since all the images are png's now.